insecure/corruptible/factory.go

```diff
@@ -194,16 +223,16 @@ func (c *ConduitFactory) processAttackerMessage(msg *insecure.Message) error {
 // RegisterAttacker is a gRPC end-point for this conduit factory that lets an attacker register itself to it, so that the attacker can
 // control it.
 // Registering an attacker on a conduit is an exactly-once immutable operation, any second attempt after a successful registration returns an error.
-func (c *ConduitFactory) RegisterAttacker(ctx context.Context, in *insecure.AttackerRegisterMessage) (*empty.Empty, error) {
+func (c *ConduitFactory) RegisterAttacker(_ context.Context, in *insecure.AttackerRegisterMessage) (*empty.Empty, error) {
```

Comment:
do we still need to pass the context even though it's not used anymore by `RegisterAttacker()`?